packages/web-components/fast-foundation/src/menu/menu.spec.md

```diff
@@ -150,12 +123,12 @@ Nested menu:
     <fast-menu-item>Menu item two</fast-menu-item>
     <fast-menu-item>
         Menu item three
-        <fast-menu slot="menu">
+        <fast-menu slot="submenu">
```

Comment:
@scomea could we do what we're doing in tree-item template and automatically set this slot if we are nested within a `fast-menu-item`? Adding the slot would work, but the nesting would also "just work" and map to existing patterns of handling this with our own controls.

Reply:
There were some timing issues that made this trickier than initially expected, but yes, this works now.